plaid/auth_test.go

```diff
@@ -22,4 +22,5 @@ func TestGetAuth(t *testing.T) {
 	}
 	accountsResp, _ := testClient.GetAccountsWithOptions(tokenResp.AccessToken, options)
 	assert.Equal(t, len(accountsResp.Accounts), 1)
+	assert.NotNil(t, accountsResp.Item)
```

Comment:
Two testing notes:

* We would also want to check `authResp.Item` above (around line 17)

* Because `Item` isn't a pointer, I don't believe this is a meaningful test. As an example, see [here](https://play.golang.org/p/FTdAyUNSGpf). What we could do instead is:
```suggestion
	assert.NotEqual(t, Item{}, accountsResp.Item)
```
Or alternatively, you could test that `accountsResp.Item.ItemID` is a non-empty string.
(Note that the asserts above are incorrect as well, It looks like this issue is currently tracked in #62.)

Reply:
Thanks for the feedback! Pushed the updates to auth_test.go along with the issues addressed in #62.